src/section/section.tsx

```diff
@@ -67,15 +81,24 @@ export const Section = ({
             </GridColumn>
```

Comment:
do we still need a special case for snakeHead?

Reply:
yes because snakeHead is putting the tail as well